src/EditorFeatures/CSharpTest/AddFileBanner/AddFileBannerTests.cs

```diff
@@ -149,7 +149,8 @@ class Program2
 @"
 <Workspace>
     <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"">
-        <Document>[||]
+        <Document>$$
+
```

Comment:
hrmm... these newlines are unintentional.  not sure what happened there...  will fix.